Testscripts/Linux/SetupRDMA.sh

```diff
@@ -606,8 +606,29 @@ function Main() {
 		Verify_File $benchmark_bin
 	fi
 
+	if [ $mpi_type != "mvapich" ]; then
+		currentDir=$(pwd)
+		cd ~
+		LogMsg "Proceeding OSU MPI Benchmark test installation"
+		LogMsg "Downloading mpi-benchmarks from $osu_mpi_benchmark"
+		wget $osu_mpi_benchmark
+		tar_filename=$(echo $osu_mpi_benchmark | rev | cut -d'/' -f1 | rev)
+		tar xvzf tar_filename
+		LogMsg "Untarred $tar_filename"
+		cd ${tar_filename%.*.*}
+
+		LogMsg "Running configuration and installing"
+		./configure --prefix=$(pwd) && make && make install
```

Comment:
OMB configure scripts detects and use mpicc / mpicxx from environment. It will be a good idea to set this explicitly as CC/CXX during config time, and will help in avoiding conflicts with other MPIs that are already installed (if any)

Reply:
Added